compiler/parser/ztests/unicode-identifier.yaml

```
@@ -0,0 +1,6 @@
+zed: 'yield тест'
+
+input: '{"тест": "значение"}'
```

Nit: No need to quote these.

```suggestion
zed: yield тест

input: '{тест: "значение"}'
```